LibreNMS/Data/Store/Rrd.php

```diff
@@ -468,15 +491,10 @@ public function buildCommand(string $command, string $filename, array $options =
             }
         }
 
-        if ($this->rrdcached &&
-            ! ($command == 'create' && version_compare($this->version, '1.5.5', '<')) &&
-            ! ($command == 'tune' && version_compare($this->version, '1.5', '<'))
-        ) {
+        if ($this->useRrdCached($command)) {
             // only relative paths if using rrdcached
             $filename = str_replace([$this->rrd_dir . '/', $this->rrd_dir], '', $filename);
```

Comment:
This feels odd to leave here if you are extacting a method to "fixRrdCachedOptions".  I'd say either move this into the method or inline the method.

Reply:
I've just pushed another update which feels more odd to me, but it moves the filename rename into the fixRrdCachedOptions function, which is what I think you are suggesting above.